numba/cuda/cudadrv/nvvm.py

```diff
@@ -501,6 +539,14 @@ def llvm_to_ptx(llvmir, **opts):
         ('declare double @___numba_atomic_double_sub(double*, double)',
          ir_numba_atomic_binary.format(T='double', Ti='i64', OP='fsub',
                                        FUNC='sub')),
+        ('declare i64 @___numba_atomic_i64_inc(i64*, i64)',
+         ir_numba_atomic_inc.format(T='i64')),
+        ('declare i64 @___numba_atomic_i64_dec(i64*, i64)',
+         ir_numba_atomic_dec.format(T='i64')),
```

Comment:
Should these be called `___numba_atomic_u64_{inc, dec}`?

Reply:
Change made.